tsrc/cli/review.py

```diff
@@ -1,19 +1,83 @@
 """ Entrypoint for Review """
 
+from operator import itemgetter
+
 from .push import get_token, PushAction
 
+import ui
+
+def get_color_status(status):
+    color = ui.green
+    if status == 'failed':
+        color = ui.red
+    if status == 'pending' or status == 'created':
+        color = ui.blue
+    if status == 'skipped':
+        color = ui.darkgray
+    return color
+
+def get_info_for_merge_status(merge_request):
+    if merge_request['merge_status'] == 'can_be_merged':
```

Comment:
we use double quotes everywhere

Reply:
CI doesn't say. do you have an automatic fixing tool for that kind of boring (and useless) tasks?